docs/content/_navigation.json

```diff
@@ -6,28 +6,20 @@
     "isDefaultOpen": true,
     "children": [
       {
-        "title": "Hello Dagster",
+        "title": "Overview",
+        "path": "/getting-started/overview"
+      },
+      {
+        "title": "Quickstart",
         "path": "/getting-started/hello-dagster"
       },
       {
         "title": "Installation",
         "path": "/getting-started/install"
       },
-      {
-        "title": "Create a new project",
```

Comment:
This is the one I'm the most on-the-fence on. @yuhan do you have thoughts on where this lives inside the new-Dagster-user journey?

Reply:
This could stay in GS - I'm not tied to it. It seems like the logical next step after installation would be creating a new project.